src/app/containers/ConsentBanner/Banner/index.amp.jsx

```diff
@@ -1,28 +1,19 @@
 import React, { useContext } from 'react';
-import { arrayOf, bool, shape, string } from 'prop-types';
-import * as AmpHelpers from 'react-amphtml/helpers';
+import { bool, string } from 'prop-types';
 import { ConsentBanner } from '@bbc/psammead-consent-banner';
 import { ServiceContext } from '../../../contexts/ServiceContext';
 import BannerText from './Text';
 
 const Accept = (message, onClick) => (
-  <AmpHelpers.Action events={onClick}>
-    {props => (
-      <button {...props} type="button">
-        {message}
-      </button>
-    )}
-  </AmpHelpers.Action>
+  <button type="button" on={onClick}>
```

Comment:
Im baffled this works, you 100% couldnt use `on` as an attribute in react 6 months ago :L

Reply:
https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html
>Note: attributes starting with on are not passed through as an exception because this could become a potential security hole.

Lol has this changed?